src/qibolab/emulator.py

```diff
@@ -0,0 +1,56 @@
+import pathlib
+
+from qibolab.channels import ChannelMap
+from qibolab.instruments.simulator.pulse_simulator import PulseSimulator
+from qibolab.platform import Platform
+from qibolab.serialize import load_qubits, load_runcard, load_settings
+
+
+def create_runcard_emulator(runcard_folder: str, pulse_simulator: PulseSimulator):
```

Comment:
Maybe we could simplify the name:
```suggestion
def create_emulator(runcard_folder: str, pulse_simulator: PulseSimulator):
```

Reply:
Fixed renaming create_emulator in 76d5cb5
Fixed platform folders in e035cb3